### PR TITLE
[seeder] refactor: router pattern

### PR DESCRIPTION
**Pull request by moonlitgrace**

### CODE REVIEW

File: seeder/src/renderer/router.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+			{
+				path: "/get-anime-urls",
+				component: lazy(() => import("@routes/anime"))
+			},
+			{
+				path: "/get-anime-themes",
+				component: lazy(() => import("@routes/anime/themes"))
+			}
+		]
+	}
```
Comment (baseplate-admin) [inline seeder/src/renderer/router.ts:42]: Very unncessary. Make it in `main.tsx`

**Review by baseplate-admin — CHANGES_REQUESTED** [review]

File: seeder/src/renderer/router.ts (hunk 1 above)
Comment (moonlitgrace) [reply seeder/src/renderer/router.ts:42]: THis might become become huge you know that right?

Comment (baseplate-admin) [reply seeder/src/renderer/router.ts:42]: Yes. i know and thats why i said so

Comment (moonlitgrace) [reply seeder/src/renderer/router.ts]: Resolved on https://github.com/coreproject-moe/CoreProject/pull/3918/commits/012dc073d3491ac68bd5cde187e2c39999b55c92

**Pull request merged**